data-raw/classifiers.R

```diff
@@ -0,0 +1,37 @@
+#!/usr/env Rscript
+
+# This script prepares classifier objects used in the package:
+# classifier.ktsp_unweighted
+# classifier.rf_weighted
+# To run this script, you must have appropriate credentials set up
+# to access the private S3 bucket s3://data-lab-mb-ssp
+
+library(optparse)
+option_list <- list(
+  make_option(
+    c("-p", "--aws_profile"),
+    type = "character",
+    default = "",
+    help = "Name of AWS profile to use for syncing baseline.rds"
+  )
+)
+opt <- parse_args(OptionParser(option_list = option_list))
+
+# Download file
+temp_file <- tempfile()
```

Comment:
It looks like you are using this as a temp directory to copy over the `baseline.rds` file, so I might be more specific here and use `tempdir()`. 
```suggestion
temp_dir <- tempdir()
```